### PR TITLE
updated teamspeak icon

### PR DESCRIPTION
**Pull request by jorgeamadosoria**
teamspeak

**Issue:** #6538 

Teamspeak has a commentary about the icon not being able to be used as monochrome. This PR is waiting reply.

### CODE REVIEW

File: svgo.config.js [hunk 1]
```diff
@@ -8,7 +8,7 @@ module.exports = {
           // customize options
           convertPathData: {
             // 3 decimals of precision in floating point numbers
-            floatPrecision: 3,
+            floatPrecision: 4,
```
Comment (adamrusted) [inline svgo.config.js:11]:
```suggestion
            floatPrecision: 3,
```

Please restore the floatPrecision to 3 in the svgo config before we merge this.

**Comment by adamrusted** [comment]
@jorgeamadosoria - looks like the thread on the TeamSpeak Community forum is stale. Do we have any contact at TeamSpeak that might be able to approve/deny this icon?

**Comment by chrisTeamSpeak** [comment]
I forwarded the request. We will reply here then :)

**Comment by syoxTS** [comment]
Hey there, I'm from the Marketing Department at TeamSpeak, here to shed some light on the situation.

As you might be aware, we currently don't have a media pack or brand guidelines available on our website, which makes things a bit tricky. The forum thread you came across is a bit outdated, and since then, we've made some changes. We do allow the use of monochrome versions of our logo, but we are striving for consistency across the board, with everyone using the same version.

In the GitHub conversation, the presented icon is quite close to what we're looking for, but there's a minor issue we'd like to address. The headset and speech bubble in the icon aren't fully filled out, which is what we would like to go for.

To help you better understand what I mean, I've attached an example of the ideal logo/icon. 
If you have any questions or concerns, feel free to let me know, and I'll get back to you promptly.

Thanks!

ts_mono_white

**Comment by adamrusted** [comment]
> In the GitHub conversation, the presented icon is quite close to what we're looking for, but there's a minor issue we'd like to address. The headset and speech bubble in the icon aren't fully filled out, which is what we would like to go for.

Thanks for the response @syoxTS!
Are you happy for our contributors to just create this icon from the SVG on the website, and run it by you for approval before we include it? In an ideal world, anything we create here should come from an official source, either an SVG hosted on a website, or in an official code repository somewhere.

**Comment by syoxTS** [comment]
Hey, @adamrusted!

Thanks for sharing the preview of the icon. 

Yes, I'm happy for your contributors to proceed with creating the SVG file for the icon. Once it's ready, please run it back for approval, and then you can move forward with releasing/updating it.

Regarding the sourcing of logos/assets, I completely agree that it's essential to ensure everything comes from official sources. I'll discuss the possibility of creating a repository for our logo and other official assets on our GitHub page with the team while we're working on our website. This way, it will be easily accessible for anyone working on projects and other things while also having an overview of what logos and variants they can use.

Thank you again for your efforts and for reaching out. I'm truly sorry for the delay and the time it took us to respond. I did check and it appears you guys had reached out to us previously via the TS3 SDK & Bizdev E-Mail. In the future, please contact marketing@teamspeak.com or reach out to us on Social Media (@teamspeak) if we don't see logo, branding, or asset requests on GitHub.

Thanks! :)

**Comment by adamrusted** [comment]
> Once it's ready, please run it back for approval, and then you can move forward with releasing/updating it.

I've uploaded a version which passes our linter to this PR, if you'd be happy to review?

> Regarding the sourcing of logos/assets

I've set our source to be the homepage for now. Once an official repo is up and running we can always update it!

**Comment by syoxTS** [comment]
Hey there,

Checked the title, hex, source, and the logo.

Looks good to go, @adamrusted.

Thanks again! Have a good one :)

**Comment by adamrusted** [comment]
Can another @simple-icons/icon-review member please review this one, given @jorgeamadosoria and myself have worked on this PR now?

**Review by LitoMore — APPROVED** [review]

**Pull request merged**